README.md

```diff
@@ -95,6 +95,8 @@ Notes:
 
 - If you are halfway through typing a command but change your mind, hit **alt-#** to add a `#` at the beginning and enter it as a comment (or use **ctrl-a**, **#**, **enter**). You can then return to it later via command history.
 
+- To avoid a command being written to your history file, you can prefix it with a space ` export AWS_SECRET_KEY=XXXXX`. This is useful if you work with API keys or passwords on the command line.
+
 - Use `xargs` (or `parallel`). It's very powerful. Note you can control how many items execute per line (`-L`) as well as parallelism (`-P`). If you're not sure if it'll do the right thing, use `xargs echo` first. Also, `-I{}` is handy. Examples:
```

Comment:
This is only true (for bash, at least) when `HISTCONTROL=ignorespace`  (or `=ignoreboth`)